Nightly release auto nvbug

albert17 opened the pull request:
- Need to figure out what to add in points 5.
- Need to set proper bug configuration (ARB, CC, Type, Status, ...)

jperez999 commented on ci/nightly.py:
```diff
@@ -0,0 +1,47 @@
+    ["3", "Expected publishing date and time", "COMPLETED"],
+    ["4", "QA results", args.qa],
+    ["5", "Security scan results", "??"],
+    ["6", "SWIPAT Approval", "??"],
```
the following link is the nvbug that gives us SWIPAT approval, so you can add it in: https://nvbugswb.nvidia.com/NvBugs5/SWBug.aspx?bugid=3267056&cmtNo=

albert17 commented:
@jperez999

Added SWIPAT link.
Changed Table from txt to html.
Added automated Bug Creation.

Parameters allow to pass container name and log output.

Need to set proper bug configuration

benfred commented:
@albert17 @jperez999 can we close this?

albert17 commented:
@benfred @jperez999 Yes we can. I already discussed with Julio that it is better to get this out of here.
